README.md

```diff
@@ -315,6 +315,22 @@ Test.ps1 | This is the PowerShell script to test that the uploaded ISO matches t
 Feel free to modify these files to suit your specific needs.
 General changes that you want to contribute back via a pull request are much appreciated.
 
+## Default GreenlakeAdmin password
+
+These scripts support one of two ways of securing the default GreenlakeAdmin account. This is the built-in Administrator account that has been renamed.
+
+1. Statically defined password in the user-data file.
```

Comment:
Does "User defined password" sound better than "Statistically defined"?

Reply:
Doesn't matter to me. What ever would be easier to understand by the consumer of the documentation.